scripts/deploy_wallet_detector.js

```diff
@@ -0,0 +1,63 @@
+// ///////////////////////////////////////////////////////////////////
+// Script to deploy a new ArgentWalletDectector.
+//
+// To deploy a new ArgentWalletDectector:
+// ./execute_script.sh deploy_wallet_detector.js <network>
```

Comment:
I get a `Error: Cannot find module '/Users/Elena/Source/argent-contracts/NO-COMPILE'` error when running this

Reply:
Good catch, there was a typo in the execute_script.sh script.